std/fs/write_json.ts

```diff
@@ -17,11 +34,7 @@ export async function writeJson(
   let contentRaw = "";
 
   try {
-    contentRaw = JSON.stringify(
-      object,
-      options.replacer as string[],
-      options.spaces,
-    );
+    contentRaw = createJsonWithEol(object, options);
```

Comment:
```suggestion
   contentRaw = format(JSON.stringify(
       object,
       options.replacer as string[],
       options.spaces,
   ), eol);
```

Reply:
This suggestion is missing the addition of an EOL at the end of file.

Reply:
It also would result in an error since `eol` is undefined in this scope.